Sources/TuistLoader/ProjectDescriptionHelpers/CachedProjectDescriptionHelpersBuilderFactory.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import Path
+import TuistSupport
+import XcodeGraph
+
+/// CachedProjectDescriptionHelpersBuilderFactory
+///
+/// is a wrapper on top of `ProjectDescriptionHelpersBuilderFactory` that adds an in-memory cache
```

Comment:
I don't think we ever use the `ProjectDescriptionHelpersBuilderFactory` by itself – should we cache the builder directly in the `ProjectDescriptionHelpersBuilderFactory` since we don't need the distinction between cached and non-cached factory?

Reply:
You are right. This may look like an overhead
I did it on the same principle as it was done with `CachedManifestLoader` and `CachedContentHasher`

I can move the current logic to `ProjectDescriptionHelpersBuilderFactory` if you see fit

Reply:
You are right. This may look like an overhead
I did it on the same principle as it was done with CachedManifestLoader and CachedContentHasher

I move the current logic to ProjectDescriptionHelpersBuilderFactory if you see fit

Reply:
Moved